docs/reference/glossary.rst

```diff
@@ -0,0 +1,56 @@
+Glossary
+========
+
+There are a variety of terms used in the documentation and throughout the
+library. Here is an overview:
+
+An action
+---------
+
+An **action** is either :code:`C` or :code:`D`.
+You can access these actions as follows but should not really have a reason to::
+
+    >>> import axelrod as axl
+    >>> axl.Actions.C
+    'C'
+    >>> axl.Actions.D
+    'D'
+
+A play
+------
+
+A **play** is a single player choosing an **action**.
+In terms of code this is equivalent to::
+
+    >>> p1, p2 = axl.Cooperator(), axl.Defector()
+    >>> p1.play(p2)  # This constitues two 'plays' (p1 plays and p2 plays).
+
+A turn
+------
+
+A **turn** is a 1 shot interaction between two players. It is in effect a
+composition of two **plays**.
```

Comment:
Each turn has four possible sets of plays: `(C, C)`, `(C, D)`,  `(D, C)`, or  `(D, D)`.